homeassistant/components/webostv/config_flow.py

```diff
@@ -98,6 +98,8 @@ async def async_step_pairing(self, user_input=None):
             except WEBOSTV_EXCEPTIONS:
                 errors["base"] = "cannot_connect"
             else:
+                await self.async_set_unique_id(client.hello_info["deviceUUID"])
```

Comment:
We always want to prefer manually started flows.
```suggestion
                await self.async_set_unique_id(
                    client.hello_info["deviceUUID"], raise_on_progress=False
                )
```